### PR TITLE
Use cozy-bootstrap for the error pages

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

File: assets/images/desert.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (Crash--) [inline assets/images/desert.svg:24]: Are SVGs optimized before being served?

Comment (nono) [reply assets/images/desert.svg:24]: nope, we don't any tool currently to process the stack assets

File: assets/templates/error.html [hunk 2]
```diff
@@ -1,40 +1,54 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="{{.Locale}}">
   <head>
     <meta charset="utf-8">
-    <title>{{.Title}}</title>
+    <meta http-equiv="refresh" content="3600">
```
Comment (Crash--) [inline assets/templates/error.html:5]: I guess you hope sometimes, after a refresh it works again? :D

File: assets/templates/instance_blocked.html [hunk 3]
```diff
@@ -1,29 +1,31 @@
...
+      <header class="wrapper-top">
+        <a href="https://cozy.io/" class="btn p-2 d-sm-none">
+          <img src="{{asset .Domain "/images/logo-dark.svg"}}" alt="Cozy Cloud" class="logo" />
+        </a>
+      </header>
+      <div class="d-flex flex-column align-items-center mb-md-3">
+        <img src="{{asset .Domain "/images/forbidden.svg"}}" alt="" class="illustration mb-3" />
+        <h1 class="h4 h2-md mb-3 text-center">{{t "Error Message instance blocked"}}</h1>
+        {{if .Reason}}<p class="text-center mb-2">{{.Reason}}</p>{{end}}
+        <p class="text-center">{{t "Error Contact instance blocked" }} <a href="mailto:contact@cozycloud.cc">contact@cozycloud.cc</a>.</p>
```
Comment (Crash--) [inline assets/templates/instance_blocked.html:26]: We don't want to use the support address defined in the context?

File: web/auth/auth.go [hunk 4]
```diff
@@ -59,12 +59,13 @@ func Home(c echo.Context) error {
...
-				"Locale":      instance.Locale,
-				"Favicon":     middlewares.Favicon(instance),
-				"CozyUI":      middlewares.CozyUI(instance),
+				"Domain":       instance.ContextualDomain(),
+				"ContextName":  instance.ContextName,
+				"Locale":       instance.Locale,
+				"Title":        instance.TemplateTitle(),
+				"ThemeCSS":     middlewares.ThemeCSS(instance),
+				"Favicon":      middlewares.Favicon(instance),
+				"SupportEmail": "contact@cozycloud.cc",
```
Comment (Crash--) [inline web/auth/auth.go:71]: Shouldn't we use the context email address?

File: web/auth/auth.go [hunk 5]
```diff
@@ -55,6 +58,9 @@ func renderError(c echo.Context, code int, msg string) error {
 // Else, it redirects to its home application (or onboarding)
 func Home(c echo.Context) error {
 	instance := middlewares.GetInstance(c)
+	if 1 == 1 {
+		return echo.NewHTTPError(404, "blah")
+	}
```
Comment (Crash--) [inline web/auth/auth.go]: blabla. You are talkative with yourself  :D

File: web/middlewares/user_agent.go [hunk 6]
```diff
@@ -79,9 +79,10 @@ var rules = []browserRule{
 	// We don't support Firefox before version 52, except on iOS where the
 	// webkit engine is used on the version numbers are not the same.
 	{
-		name:       Firefox,
-		iPhone:     notIphone,
-		minVersion: 52,
+		name:   Firefox,
+		iPhone: notIphone,
+		// minVersion: 52,
+		minVersion: 99,
```
Comment (Crash--) [inline web/middlewares/user_agent.go]: guess it's removed later

File: assets/templates/instance_blocked.html (hunk 3 above)
Comment (nono) [reply assets/templates/instance_blocked.html:26]: I was like that before, and I would like to talk with Zoé before making the change

**Pull request merged**